slides/workshop_rap_for_managers_basics.qmd

```diff
@@ -0,0 +1,343 @@
+:::
+
+::: {.column width="20%"}
+Slido:
+
+![](images/RFM-slido.png){fig-align="center"}
+:::
+
+::: {.column width="40%"}
+:::
+
+# What is RAP?
+
+::: {.fragment .fade-up}
+<span style="color:#f67400;">**R**</span>eproducible <span style="color:#f67400;">**A**</span>nalytical <span style="color:#f67400;">**P**</span>ipelines are simply about using automation to our advantage when analysing data, and ensuring we are transparent when we do!
+:::
+
+::: {.fragment .fade-up}
+We already have *analytical pipelines* and have done for many years. The aim of RAP is to:
+:::
+
+::: {.fragment .fade-up}
+-   Automate all of the parts that we can
+:::
+
+::: {.fragment .fade-up}
+-   Increase efficiency and accuracy
+:::
+
+::: {.fragment .fade-up}
+-   Create a clear audit trail to allow analysis to be easily reused if needed
+:::
+
+# What does RAP look like?
+
+# RAP in "real life" - case studies
+
+::: {.fragment .fade-up}
+Implementing RAP best practices will free up time for us to focus on the parts of our work where human input adds true value.
+:::
+
+::::::::: columns
+:::: {.column width="33%"}
+::: {.fragment .fade-up}
+> *"The previous process with the two Excel workbooks would take a couple of months to run, whereas now it can be run in a matter of hours."*\
+> **- Matt Jago, Ops & infrastructure**
+:::
+::::
+
+:::: {.column width="33%"}
+::: {.fragment .fade-up}
+> *"\[The improvements have\] reduced the human errors and also mean analysts and LA colleagues’ time can be spent on more valuable tasks."*\
+> **- Family hubs research and analysis team**
+:::
+::::
+
+:::: {.column width="33%"}
+::: {.fragment .fade-up}
+> *"The script itself is not complicated, but it removes the risk of human error and reduces the production time to minutes. The time saved can then be dedicated to further quality assurance."*\
+> **- RAAC response analyst**
+:::
+::::
+:::::::::
+
+# Improving pipeline efficiency: before RAP
+
+::: {.column width="10%"}
+:::
+
+::: {.column width="80%"}
+![](images/RAP-flowchart-white-before.png)
+:::
+
+::: {.column width="10%"}
+:::
+
+# Improving pipeline efficiency: after RAP
+
+::: {.column width="10%"}
+:::
+
+::: {.column width="80%"}
+![](images/RAP-flowchart-white-after.png)
+:::
+
+::: {.column width="10%"}
+:::
+
+# What it means for managers
+
+On our [RAP expectations page](https://dfe-analytical-services.github.io/analysts-guide/RAP/rap-expectations.html#analyst-managers), we have pulled out the expectations for analyst managers from the [cross-government RAP strategy](https://analysisfunction.civilservice.gov.uk/policy-store/reproducible-analytical-pipelines-strategy/), which include:
+
+> -   build extra time into projects to adopt new skills and practices where appropriate
+> -   learn the skills they need to manage software
+> -   evaluate RAP projects within organisations to understand and demonstrate the benefits of RAP
+> -   mandate their teams use RAP principles whenever possible
+
```

Comment:
I think there's something in the general AF expectations for analysts, the code of practice for stats and gss competency framework around this stuff too which you could pull out either here or earlier - it's not nice to have, it's what is expected for our roles. 
Could be worth a look to dig those out, or ask Cam as I think he referenced them in a slide pack he did at a DMM last year